src/main/resources/SnakemakeBundle.properties

```diff
@@ -254,6 +254,12 @@ notifier.msg.framework.by.snakefile.title=Snakemake framework detected
 notifier.msg.framework.by.snakefile.action.configure=Configure Framework...
 notifier.msg.framework.by.snakefile=Snakefile was found in ''{0}''.
 
+#######################
+# Banner
+BANNER.msg=Snakemake support is disabled. Not all features are available.
```

Comment:
Please be more precisely. `BANNER` is to general. Probably will have other banners in future. E.g use prefix `banner.smk.framework.not.configured'